products/eap/_common/product.yml

```diff
@@ -24,9 +24,26 @@ guides:
 default_download_artifact_type: installer
 description: 'An application platform for hosting your apps that provides an innovative modular, cloud-ready architecture, powerful management and automation, and world class developer productivity.'
 downloads:
+  7.0.0.Beta:
+    description: Major release of JBoss EAP 7.x series
+    tag_line: Featuring support for Java EE 7, high performance Undertow web server, port reduction, and more. Try it today!
+    release_date: 2015-12-16
+    assets:
+      zip:
+        size: 150mb
+        name: Zip File
+      maven_repo:
+        size: 250mb
+        name: Maven Repository
+      quickstarts:
+        size: 9mb
+      source:
+        size: 20mb
+      javadoc:
+        size: 173mb
   7.0.0.Alpha:
     description: Major release of JBoss EAP 7.x series
-    tag_line: Featuring Java EE 7 Support. Try it today!
+    tag_line: Featuring support for Java EE 7, high performance Undertow web server, port reduction, and more. Try it today!
```

Comment:
Ah, just seen that you fixed the Beta tagline too. Although superfluous, this change is fine.